blueocean-rest-impl/pom.xml

```diff
@@ -43,6 +43,11 @@
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>credentials</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Should be moved under `dependencyManagement` in parent pom and declared here without version.